src/mg/PAMI/Logger/Log4PhpAdapter.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace PAMI\Logger;
+
+use Logger;
+use LoggerLevel as Log4PhpLevel;
+use LoggerLevel;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel as PsrLevel;
+
+class Log4PhpAdapter implements LoggerInterface
```

Comment:
Given the fact that the log4php project hasn't had any commits in the past two years, I don't see much use in creating a PR for this.

Splitting this adapter out into a separate package on the other hand, would certainly be possible though.